### ISSUE TITLE
Unhandled exception: [Enter a title]

### ISSUE THREAD

**Issue by Istrom** [remyroy/CDDA-Game-Launcher#334]
* Description: [Enter what you did and what happened]
* Version: 1.3.22
* OS: Windows-7-6.1.7601-SP1 (64-bit)
* Type: `<class 'FileNotFoundError'>`
* Value: [Errno 2] No such file or directory: 'C:\\Users\\derek\\Downloads\\CO.AG SoundPack (v1.1) (02-Mar-19).zip'
* Traceback:
```
  File "cddagl\ui.py", line 4567, in install_new
  File "zipfile.py", line 1090, in __init__

```

**Comment by Istrom** [comment]
I tried to install the CO.AG soundpack, and the launcher immediately crashed.

**Comment by remyroy** [comment]
Did you download the soundpack with your browser? Did you select the path that pointed to the downloaded file afterwards?

**Comment by Istrom** [comment]
Sorry for the late response!

I had selected the path through the interface for the launcher to have it
installed by the launcher, that would always immediate crash the whole
executable.

I was able to use the separate download link from the same interface to do
manual file downloads, and manual sound pack installs. I'm currently
running the 2 Ch sounds pack with CO.AG music, and it works great.

I hope this helped.

Derek
